src/auth/User.cc

```diff
@@ -69,52 +68,7 @@ Auth::User::absorb(Auth::User::Pointer from)
     notes.appendNewOnly(&from->notes);
```

Comment:
Here, we ignore any same-key annotations found `from`. In merge() below, we use same-key annotations in `from` to replace the ones in `this`. If this difference is intentional, please add a C++ comment to highlight and explain the difference in merging two containers (`notes` and `ipList`).